README.md

```diff
@@ -120,6 +120,17 @@ and attributes where this model code came from, if any. This also helps to
 show case what changes we have done to make it performant on TPU. The original
 version is not expected to be run.
 
+## Run huggingface transformer models
+Torchprime supports run with huggingface models by taking advantage of `tp run`. To use huggingface models, you can clone [huggingface/transformers](https://github.com/huggingface/transformers) under torchprime. This allows you to pick any branch or make code modifications for experiment:
```

Comment:
nit: the rest of this file is wrapped at 80 characters for readability. I use a vscode extension called "rewrap" to do this automatically.